src/models/db/accounts.js

```diff
@@ -30,7 +30,11 @@ function * _upsertAccount (account, options) {
     existingAccount.setData(account)
     yield existingAccount.save(options)
   } else {
-    yield Account.create(account, options)
+    if (!account.balance) {
+      yield Account.create(_.merge({}, account, {balance: 0}), options)
+    } else {
+      yield Account.create(account, options)
+    }
```

Comment:
I'm having trouble getting the modified db script to work with knex for several reasons.

1. knex v. 0.10 won't properly insert default values, so I can't set `default 0` for the `balance` column in the `accounts` table in the db script
2. knex v. 0.11 fixes the above issue, but breaks oracle: `Knex:warning - Pool2 - Error: ORA-00923: FROM keyword not found where expected`. The root cause is unknown, and could take some time to track down.
3. Even if the above two issues were fixed, we would still need this code for sqlite, since sqlite doesn't support inserting default values.

Since this is a hotfix for the release branch, I'm inclined to take the easy route and go with the above code, so that I have time to look into the oracle failures on knex v. 11 in more detail. What do you think @clark800?